include/mo.php

```diff
@@ -53,7 +55,11 @@ public function import_from_db( $lang ) {
 		}
 
 		foreach ( $strings as $msg ) {
-			$this->add_entry( $this->make_entry( $msg[0], $msg[1] ) );
+			$entry = $this->make_entry( $msg[0], $msg[1] );
```

Comment:
Is checking `'' !== $msg[0]` enough?

Reply:
I prefer to use `make_entry()` because of [this](https://github.com/WordPress/WordPress/blob/fe6b65c44b929ee5583f6146d92388ef43061f9a/wp-includes/pomo/mo.php#L322-L329). Do you think I'm going too far?

Reply:
Oh I see, maybe a bit but we never know 😅